app/views/case_contacts/form/expenses.html.erb

```diff
@@ -97,6 +97,17 @@
     <% end %>
 
     <div class="actions mb-10 case-contacts-form-buttons">
+      <div class="checkbox-style pt-1">
+        <%= form.fields_for :metadata do |metadata_form| %>
+          <%= metadata_form.check_box :create_another, class: "form-check-input" %>
+          <%= metadata_form.label :create_another, class: "form-check-label" do %>
+            Create Another
+            <i class="lni lni-question-circle" data-toggle="tooltip" data-placement="top"
+              title="Start a new contact for the same case(s) after submitting.">
+            </i>
+          <% end %>
+        <% end %>
+      </div>
```

Comment:
styling looks off on my screen 
![image](https://github.com/user-attachments/assets/7ef330d1-0323-4400-bb3c-15ea9bfccd00)

Reply:
dang! yeah, that really bothers me, will try to fix.